spring-integration-azure/spring-integration-eventhubs/src/test/java/com/microsoft/azure/spring/integration/eventhub/converter/EventHubMessageConverterTest.java

```diff
@@ -26,4 +47,68 @@ protected EventData getInstance() {
     protected Class<EventData> getTargetClass() {
         return EventData.class;
     }
+
+    private static class MyEventHubMessageConverter extends EventHubMessageConverter {
```

Comment:
Why do we need this subclass here if it's simply calling super class?

Reply:
hi @saragluna Must use sub class to access the method to test.